src/mapOfPeople.js

```diff
@@ -40,7 +40,14 @@
  * @return {Map}
  */
 function mapOfPeople(people) {
-  // write code here
+  const peopleAtTheRoom = new Map();
+  people.forEach((item, seconds) => {
+    if (!peopleAtTheRoom.delete(item)) {
```

Comment:
short and smarty but hard to understand -> hard to maintain. Do not do side effects inside if condition.
```javascript
    const hasBeenRemoved = peopleAtTheRoom.delete(item);

    if (!hasBeenRemoved) {
```
It is more explicite